docs/getting-started/quick-start.md

```diff
@@ -184,9 +184,35 @@ fun RootContent(component: RootComponent, modifier: Modifier = Modifier) {
 }
 ```
 
+### Child Stack with SwiftUI
+
+```swift
+struct RootView: View {
+    private let root: RootComponent
+    
+    @ObservedObject
+    private var childStack: ObservableValue<ChildStack<AnyObject, RootComponentChild>>
```

Comment:
Let's copy the explanation of what `ObservableValue` is from [here](https://arkivanov.github.io/Decompose/component/overview/#what-is-observablevalue). Otherwise it will be unresolved and it's not obvious where it comes from.